Backpack/src/main/java/net/skyscanner/backpack/starrating/BpkStarRating.kt

```diff
@@ -38,6 +39,18 @@ open class BpkStarRating @JvmOverloads constructor(
   starSize = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingBase)
 ) {
 
+  override fun onInitializeAccessibilityNodeInfo(info: AccessibilityNodeInfo) {
+    super.onInitializeAccessibilityNodeInfo(info)
+    val text = getAccessibilityText()
+    if (text != null) {
+      if (info.contentDescription != null) {
+        info.contentDescription = "$text ${info.contentDescription}"
```

Comment:
This is an annoying hack - i couldn't get both an announcement and the standard content description being announced (at least not pre-30) - pre-30 without a content description nothing was announced, so use the accessibility text and append the content description if available, otherwise only announce the accessibility text

Reply:
Might be worth adding the above or similar as a comment alongside this code so that we have a record for the future as why its like this?